common/stream.cpp

```diff
@@ -52,6 +52,16 @@ SeekableReadStream *ReadStream::readStream(uint32 dataSize) {
 	return new MemoryReadStream((byte *)buf, dataSize, DisposeAfterUse::YES);
 }
 
+Common::String ReadStream::readString(char terminator) {
```

Comment:
If you make terminator parametrisable, please make sure you don't add \0 into the string

Reply:
Shouldn't be a problem. If in the rare case there's a different string terminator, like '$' for DOS strings, then there wouldn't be any 0's to begin with. Anyway, this all looks good, and I'll be fine with it being merged in.